frog/imports/ui/StudentView.jsx

```diff
@@ -99,6 +97,8 @@ const StudentView = ( { user, sessions, products } ) => { return(
       products={products} />
     <SessionList 
       sessions={sessions} />
+      sessions={sessions}
+      curSessionId={curSession && curSession._id} />
```

Comment:
I can't find where curSession is defined. Get error 
```
StudentView.jsx:101 Uncaught ReferenceError: curSession is not defined
``` 
when running it in browser. Finally `/>`  seems to close nothing